schema.sql

```diff
@@ -0,0 +1,287 @@
+CREATE TABLE
+  account (
+    id VARCHAR PRIMARY KEY,
+    total_donations_usd DECIMAL(10, 2),
+    total_matching_pool_allocations_usd DECIMAL(10, 2),
+    donors_count INT
+  );
+
+CREATE TABLE
+  list (
+    id VARCHAR PRIMARY KEY,
+    owner_id VARCHAR NOT NULL,
+    name VARCHAR NOT NULL,
+    description TEXT,
+    default_registration_status ENUM(
+      'Pending',
+      'Approved',
+      'Rejected',
+      'Graylisted',
+      'Blacklisted'
+    ) NOT NULL,
+    FOREIGN KEY (owner_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  list_admin (
+    list_id INT NOT NULL,
+    admin_id VARCHAR NOT NULL,
+    PRIMARY KEY (list_id, admin_id),
+    FOREIGN KEY (list_id) REFERENCES list (id),
+    FOREIGN KEY (admin_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  list_registration (
+    id SERIAL PRIMARY KEY,
+    registrant_id VARCHAR NOT NULL,
+    status ENUM(
+      'Pending',
+      'Approved',
+      'Rejected',
+      'Graylisted',
+      'Blacklisted'
+    ) NOT NULL,
+    submitted_at TIMESTAMP NOT NULL,
+    updated_at TIMESTAMP,
+    registrant_notes TEXT,
+    admin_notes TEXT,
+    tx_hash VARCHAR,
+    FOREIGN KEY (registrant_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  pot_factory (
+    id VARCHAR PRIMARY KEY,
+    owner_id VARCHAR NOT NULL,
+    deployed_at TIMESTAMP NOT NULL,
+    source_metadata VARCHAR NOT NULL,
+    protocol_fee_basis_points INT NOT NULL,
+    protocol_fee_recipient_account VARCHAR NOT NULL,
+    require_whitelist BOOLEAN NOT NULL,
+    FOREIGN KEY (owner_id) REFERENCES account (id),
+    FOREIGN KEY (protocol_fee_recipient_account) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  pot_factory_admin (
+    pot_factory_id INT NOT NULL,
+    admin_id VARCHAR NOT NULL,
+    PRIMARY KEY (pot_factory_id, admin_id),
+    FOREIGN KEY (pot_factory_id) REFERENCES pot_factory (id),
+    FOREIGN KEY (admin_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  pot_factory_whitelisted_deployer (
+    pot_factory_id INT NOT NULL,
+    whitelisted_deployer_id VARCHAR NOT NULL,
+    PRIMARY KEY (pot_factory_id, whitelisted_deployer_id),
+    FOREIGN KEY (pot_factory_id) REFERENCES pot_factory (id),
+    FOREIGN KEY (whitelisted_deployer_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  pot (
+    id VARCHAR PRIMARY KEY,
+    pot_factory_id INT NOT NULL,
+    deployer_id VARCHAR NOT NULL,
+    deployed_at TIMESTAMP NOT NULL,
+    source_metadata VARCHAR NOT NULL,
+    owner_id VARCHAR NOT NULL,
+    chef_id VARCHAR NULL,
```

Comment:
for consistency and clarity's sake, either remove this `NULL` and other instances (as it is the default), or use it consistently across all tables for any nullable columns

Reply:
my personal preference would be to remove `NULL` as I think it's easier to read